### DISCUSSION TITLE
Versione 1.5.8

### DISCUSSION THREAD

**Discussion by luigirotondi** [iisgiua/giuaschool#698] [Richieste di aiuto] [answered]
Salve, si sa quando verrà rilasciata la versione 1.5.8?

**Comment by trinko** [comment, marked as answer]
Entro la fine del mese.

**Reply by luigirotondi** [reply]
Cosa prevede la versione 1.5.8?

**Comment by trinko** [comment]
> Cosa prevede la versione 1.5.8?

-    nuova gestione note disciplinari, con annullamento formale, separazione tra nota e provvedimento disciplinare;
-    correzione e modifiche allo scrutinio finale, con inserimento della nuova certificazione delle competenze;
-    gestione RSPP e possibilità di inserirlo come destinatario degli avvisi;
-    correzioni di vari bug minori.
